tools/proxy/proxy.js

```
@@ -24,6 +29,9 @@ function makeEnv(port, host) {
   return env;
 }
 
+const webpack_tls_args = ["--server-type", "https", "--server-options-key", "./target/dev.key.pem", , "--server-options-cert", "./target/dev.cert.pem"];
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)